build.gradle.kts

```diff
@@ -614,7 +614,7 @@ tasks {
     dependsOn("spark-connector:shadowJar")
     group = "gravitino distribution"
     finalizedBy("checksumSparkConnector")
-    from("spark-connector/build/libs/${rootProject.name}-spark-connector-runtime-$version.jar")
+    from("spark-connector/build/libs/${rootProject.name}-spark-connector-runtime-3.4_$scalaVersion-$version.jar")
```

Comment:
could you use spark version instead of `3.4`?

Reply:
using `sparkMajorVersion`